Bottom: 7dffb5c05030764bf0d830377de3a957d588633d
Top:    e991bba32df9df582f8f48dea740a07b91e5e9d9
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-02 16:15:19 +0800

Refresh of autofs-5.0.7-use-MS_UNBINDABLE-for-autofs-mounts.patch

---

diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index d75e117..23cc284 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -30,6 +30,7 @@
 #include <sys/utsname.h>
 #include <netinet/in.h>
 #include <linux/fs.h>
+#include <sys/mount.h>
 
 #define MODULE_PARSE
 #include "automount.h"
